cli/commands/migrate.ts

```diff
@@ -72,7 +72,7 @@ export const migrate = async (cli: CLIEnvironment, cliArgs: CLIArgs): Promise<vo
 
   if (chainId == 1337) {
```

Comment:
Once we move the list of contracts to deploy to a config file we won't need this if/case